packages/neon/neon/lib/src/widgets/image.dart

```diff
@@ -354,19 +354,27 @@ class NeonUrlImage extends StatelessWidget {
   Widget build(final BuildContext context) {
     final account = this.account ?? NeonProvider.of<AccountsBloc>(context).activeAccount.value!;
 
+    final uri = Uri.parse(url);
+    final dataUri = uri.scheme == 'data' ? UriData.fromUri(uri) : null;
```

Comment:
```suggestion
    final dataUri = uri.data;
```
I prefer inlining this in the below check as this variable would be redundant.
Also for your information `uri.isScheme('data')` would be preferred as it does the comparison case insensitive.

Reply:
The data URI is also used in the hint, that's why I moved it outside.

Reply:
Any specific reason you don't like `uri.data` or just so the code doesn't look ugly.
I agree that there isn't any difference.

Reply:
Oh I didn't know there was a getter for it. I'll use it instead